draft-ietf-tls-tls13.md

```diff
@@ -4195,6 +4195,16 @@ Cryptographic details:
   {{ffdhe-param}})?
 
 
+## Client Tracking Prevention
+
+Clients concerned with privacy against tracking by passive observers
+SHOULD use a PSK/session ticket at most once. Servers SHOULD issue
+more than one session ticket per handshake, or issue a new session
+ticket on every resumption handshake, to assist in the privacy of the
+client while preserving the performance advantage of session
```

Comment:
I think this sounds good: this captures my original intent, states the justification, and is more specific in its guidance.